.github/workflows/beta.yml

```diff
@@ -0,0 +1,116 @@
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  GATSBY_CPU_COUNT: 2
+  GATSBY_ENV: staging
+
+jobs:
+  lint:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    environment: staging
+
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: "npm"
+
+      - name: Install Node.js dependencies
+        run: |
+          npm ci
+          npm test
+
+  release-beta:
+    needs: lint
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    environment: staging
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18.x"
+          cache: "npm"
+
+      - run: npm ci
+      - run: npm run format
+      - run: npm run test
+      - run: npm run build
+      - run: npm run build:storybook
+      - uses: olegtarasov/get-tag@v2.1
+        id: tagName
+        # For using same tag for staging and production we need to uncomment these two below lines:
+        # with:
+        #   tagRegex: "staging(.*)"
+      - name: Building docker image 🐳
+        run: docker build -t ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:latest-beta  -t ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:$GITHUB_SHA .
+
+      - name: Verify nginx image
+        run: |
+          set -e
+          docker run --rm ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:$GITHUB_SHA nginx -t
+          echo "docker image validated successfully"
+
+      - name: Pushing Image to docker hub 🐳
+        run: |
+          echo ${{ secrets.DOCKERHUB_PASSWORD }}| docker login -u ${{ secrets.DOCKERHUB_USERNAME }} --password-stdin
+          docker push  ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:latest-beta
+          docker push  ${{ secrets.DOCKHUB_ORGANISATION }}/deriv-com:$GITHUB_SHA
+
+      - name: Deploy 🚀
+        run: |
+          export NAMESPACE="deriv-com-beta"
+          export SERVICEACCOUNT_TOKEN=${{ secrets.SERVICEACCOUNT_TOKEN }}
+          export DOCKERHUB_ORGANISATION=${{ secrets.DOCKERHUB_ORGANISATION }}
+          export KUBE_SERVER=${{ secrets.KUBE_SERVER }}
+          export CA=${{ secrets.CA }}
+          git clone https://github.com/binary-com/devops-ci-scripts
+          cd devops-ci-scripts/k8s-build_tools
+          echo ${{ secrets.CA_CRT}} | base64 --decode > ca.crt
+          ./release.sh deriv-com $GITHUB_SHA
+
+      - name: Deploy to Cloudflare ☁️
+        uses: cloudflare/wrangler-action@2.0.0
+        with:
+          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+          accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+          command: pages publish public --project-name=deriv-com-pages --branch=translation-beta
+
+      - name: Cloudflare preview link ✨
+        run: echo "New beta website - http://beta.cf-pages-deriv-com.deriv.com"
```

Comment:
`beta.deriv.com` and `beta-eu.deriv.com`

Reply:
Updated this